core/field.js

```diff
@@ -768,25 +766,21 @@ Blockly.Field.prototype.setTooltip = function(_newTip) {
  * @private
  */
 Blockly.Field.prototype.getClickTarget_ = function() {
+  if (this.clickTarget_) return this.clickTarget_;
```

Comment:
@rachel-fenichel here.

Reply:
Not sure--I feel like this can vary a lot by field.  But I think letting it be the svg root here is fine in the general case, and if it comes up for other fields they can override.

Reply:
Makes sense.